docs/mkdocs.yml

```diff
@@ -71,7 +72,6 @@ nav:
     - Overview: reference/index.md
     - Client Libraries: reference/client-libraries.md
     - Security: reference/security.md
-    - reference/partner-certs.md
```

Comment:
Don't rename the file, it'll leave any public links dangling. We could use something like `mkdocs-redirect` to put a redirection in place to make it work though

https://github.com/mkdocs/mkdocs-redirects